### PR TITLE
add inverted index classifier

### PR DESCRIPTION
**Pull request by TkrUdagawa**
related to jubatus/jubatus_core#234

### CODE REVIEW

File: source/en/api_classifier.rst [hunk 1]
```diff
@@ -129,6 +131,43 @@ We show each field below:
 
         * Range: 0.0 <= ``local_sensitivity``
 
+   cosine
+     :parameter:
```
Comment (kmaehashi) [inline source/en/api_classifier.rst]: Is this correct?

File: source/en/api_classifier.rst [hunk 2]
```diff
@@ -28,6 +28,8 @@ We show each field below:
       ``"AROW"``       Use Adaptive Regularization of Weight vectors. [Crammer09b]_
       ``"NHERD"``      Use Normal Herd. [Crammer10]_
       ``"NN"``         Use an inplementation of ``nearest_neighbor``
```
Comment (kmaehashi) [inline source/en/api_classifier.rst]: For these 3 algorithms, I want to include a keyword `k-NN` (or `k-近傍法`) which is easy to understand for many people.

File: source/en/api_classifier.rst [hunk 3]
```diff
@@ -129,6 +131,43 @@ We show each field below:
...
+     :local_sensitivity:
+        Sensitivity used for caliculating scores.
+        When it is bigger, near data are weighted much more.
+        When it is 0, all data will be treated as same weight.
+        (Float)
+
+        * Range: 0.0 <= ``local_sensitivity``
+
+   euclidean
+     :parameter:
```
Comment (kmaehashi) [inline source/en/api_classifier.rst]: Is this correct?

**Comment by kmaehashi** [comment]
Link to PR: https://github.com/jubatus/jubatus_core/pull/276

**Comment by kmaehashi** [comment]
Please document the limitations:
* unlearner is not supported
* `delete_label` is not supported

File: source/en/api_classifier.rst (hunk 3 above)
Comment (TkrUdagawa) [inline source/en/api_classifier.rst]: Thanks. My mistakes.

File: source/en/api_classifier.rst (hunk 1 above)
Comment (TkrUdagawa) [inline source/en/api_classifier.rst]: Thanks. My mistakes.

File: source/en/api_classifier.rst (hunk 2 above)
Comment (TkrUdagawa) [inline source/en/api_classifier.rst]: I have added Method type column, which mentions the feature of the method (linear classification or k-Nearest Neighbor).

**Comment by kmaehashi** [comment]
:+1: LGTM!

**Pull request merged**